backend/app/Http/Controllers/UserController.php

```diff
@@ -19,7 +19,11 @@ public function store(Request $request) { // register
         $attrs = $this->validateUser($request);
 
         $user = User::create($attrs);
-        return new UserResource($user);
+        $token = $user->createToken('login')->plainTextToken;
+        return response()->json([
```

Comment:
new line before return 
```suggestion
        $token = $user->createToken('login')->plainTextToken;
        
        return response()->json([
```